apps/web/utils/server-action.ts

```
@@ -13,3 +18,26 @@ export function handleActionResult<T>(
     toastSuccess({ description: successMessage });
   }
 }
+
+export async function handleActionCall<T, E = {}>(
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)